packages/commitlint-config/index.spec.js

```diff
@@ -2,8 +2,8 @@ import lint from '@commitlint/lint'
 import cozyConfig from '.'
 
 const rules = cozyConfig.rules
-const TOO_LONG =
-  'This is an invalid sentence since it has too much character !!!!!!!!!!!!!'
+// const TOO_LONG =
+//  'This is an invalid sentence since it has too much character !!!!!!!!!!!!!'
```

Comment:
I would suggest not commenting on the code because we never know why it was commented on.